src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3462,6 +3463,62 @@ await _githubApi.Invoking(api => api.AbortMigration(migrationId))
             .WithMessage(expectedErrorMessage);
     }
 
+    [Fact]
+    public async Task UploadArchiveToGithubStorage()
+    {
+        //Arange 
+        const string org = "1234";
+        const bool isMultipart = false;
+        const string archiveName = "archiveName";
+
+        // Using a MemoryStream as a valid stream implementation
+        using var archiveContent = new MemoryStream(new byte[] { 1, 2, 3 });
+        var expectedArchiveId = "123456";
+        var jsonResponse = $"{{ \"archiveId\": \"{expectedArchiveId}\" }}";
+
+        _githubClientMock
+            .Setup(m => m.PostAsync(It.IsAny<string>(), It.IsAny<HttpContent>(), null))
```

Comment:
Let's make sure that `PostAsync` receives a `StreamContent`:

```suggestion
            .Setup(m => m.PostAsync(It.IsAny<string>(), It.IsAny<StreamContent>(), null))
```